userguide/vmware.rst

```diff
@@ -0,0 +1,115 @@
+.. index:: VMware Recommendations
+.. _VMware Recommendations:
+
+VMware Recommendations
+----------------------
+
+This section offers %brand% configuration recommendations and
+troubleshooting tips when using %brand% with a
+`VMware <https://www.vmware.com/>`__ hypervisor.
+
+#ifdef truenas
+
+ALUA **must** be used when configuring
+:ref:`iSCSI Sharing <Block (iSCSI)>` for a %brand% with
+High Availability (HA). This allows %brand% to
+:ref:`failover <Failover>` when there is an active iSCSI share.
+
+#endif truenas
+
+#ifdef freenas
+To create a new %brand% Virtual Machine (VM) in VMware, see the
+:ref:`VMware ESXi` section of this guide.
+#endif freenas
+
+When creating a new Virtual Machine (VM), be sure to install the VMware
+guest tools. VMware provides instructions to
```

Comment:
This section mixes up two subjects: running FreeNAS as a guest on VMware, and using FreeNAS/TrueNAS to host storage for VMware.  These are wildly different uses and should be separated into different sections of this chapter.